src/test/java/com/atomist/rug/cli/AbstractCommandTest.java

```diff
@@ -112,6 +112,7 @@ protected void assertSuccess(String requiredContents, String... tokens) throws E
             commandLine.add(config.getCanonicalPath());
             commandLine.add("-X");
             commandLine.add("-t");
+            commandLine.add("-l");
```

Comment:
Oh. That's unfortunate. As it stands, tests have been failing without it. I will add that flag only to tests that were failing before the change, is that okay?